clients/client-appconfig/src/protocols/Aws_restJson1.ts

```diff
@@ -326,7 +326,7 @@ export const se_CreateHostedConfigurationVersionCommand = async (
 ): Promise<__HttpRequest> => {
   const b = rb(input, context);
   const headers: any = map({}, isSerializableHeaderValue, {
-    [_ct]: input[_CT]! || "application/octet-stream",
+    [_ct]: input[_CT] || "application/octet-stream",
```

Comment:
this isn't related to idempotency tokens, but the codegen now removes this extraneous `!` assertion.